internal/cli/init.go

```diff
@@ -17,6 +17,7 @@ import (
 
 	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
 
+	hclpkg "github.com/hashicorp/waypoint/hclgen"
```

Comment:
yup, this should be `... waypoint/internal/cli/hclgen` to follow existing patterns, like datagen.